pytorch_to_returnn/torch/nn/modules/operator.py

```diff
@@ -52,30 +52,51 @@ class RandInt(Module):
 
   def create_returnn_layer_dict(self, low, high, size, dtype=None) -> Dict[str, Any]:
     dtype = dtype or "int64"
-    assert isinstance(low, int)
-    assert isinstance(high, int)
-    assert all(isinstance(sz, int) for sz in size)
-    return {"class": "rand_int", "shape": size, "maxval": high, "minval": low, "dtype": dtype}
+    if isinstance(low, Tensor):
+      low = low.type(dtype)
+      low = self._get_input_layer_name(low)
+    if isinstance(high, Tensor):
+      high = high.type(dtype)
+      high = self._get_input_layer_name(high)
+    naming = Naming.get_instance()
+    call = naming.module_call_stack[-1]
+    assert call.module.module is self
+    source = set()
+    for sz in size:
+      if isinstance(sz, Tensor):
+        for dep_tensor in naming.tensors[sz].creation_stack_call.inputs_flat:
```

Comment:
You should not access `creation_stack_call` here. You should not use any such assumptions.

You should not use `inputs_flat`.

Why do you do that anyway?